components/brave_new_tab_ui/components/default/braveToday/customize/Context.tsx

```diff
@@ -24,6 +24,8 @@ interface BraveNewsContext {
   filteredPublisherIds: string[]
   // Publishers the user is directly subscribed to
   subscribedPublisherIds: string[]
+  // Publishers to suggest to the user.
+  suggestedPublisherIds: string[]
```

Comment:
At some point we might want to make some of these items lazy-load and expose promises, like redux toolkit's query. Especially when we use the BraveNewsContext for the regular feed view. Loading all the data from all the API calls immediately on open could cause a bit of blocking.

Reply:
:+1: